inst/tutorials/bednets/bednets.Rmd

```diff
@@ -1,75 +1,211 @@
+
+```{r quiz-1}
+quiz(caption = "Parameters Qs",
+     question("What mosquito death rate (dn0) have we specified in the chunk above?",
+              answer("365"),
+              answer("0.56", correct = TRUE),
+              answer("0.24"),
+              answer("0"), allow_retry = T),
+     question("How many columns must the matrix inputs for dn0, rn, and rnm contain if we specify bed net events to be updated 5 times (i.e. we want parameters to be updated on 5 timesteps)?",
+              answer("3"),
+              answer("Depends on the number of mosquito species"),
+              answer("1"),
+              answer("5", correct = TRUE))
+)
 
-*Here's an exercise with some prepopulated code as well as `exercise.lines = 5` to provide a bit more initial room to work.*
+```
 
-Now write a function that adds any two numbers and then call it:
+Now that we've parameterised a bed net campaign, let's run the simulation! Note that we still include the step to equilibrate the parameters to a given level of transmission (`initial_EIR`).
+
+```{r set_bednets_example, exercise = TRUE}
+
+# Load the base parameter set:
+parameters <- get_parameters(overrides = list(human_population = 1000))
+
+# Load the bed net parameters into the parameter list using set_bednets()
+parameters <- set_bednets(parameters = parameters,
+                          timesteps = c(1, 4) * 365,
+                          coverages = c(0.5, 0.5),
+                          retention = 5 * 365,
+                          dn0 = matrix(c(0.533, 0.533), nrow = 2, ncol = 1),
+                          rn =  matrix(c(0.56, 0.56), nrow = 2, ncol = 1),
+                          rnm =  matrix(c(0.24, 0.24), nrow = 2, ncol = 1), 
+                          gamman =  rep(2.64 * 365, 2))
+
+# Equilibrate the model parameters to an initial EIR:
+parameters <- set_equilibrium(parameters = parameters, 
+                              init_EIR = 8)
+
+# Run the simulation
+simulation_output <- run_simulation(timesteps = 7 * 365, 
+                                    parameters = parameters)
+
+# Plot the mosquito population through time:
+simulation_output |>
+  mutate(total_mosquito_population = Sm_gamb_count + Im_gamb_count + Pm_gamb_count) |>
+  ggplot(aes(x = timestep, y = total_mosquito_population)) +
+  geom_line(linewidth = 1.2, colour = "deepskyblue2") +
+  theme_bw() +
+  labs(x = "Time (days)", y = "Adult Female Mosquito Population Size") +
+  scale_y_continuous(limits = c(0, 7500)) +
+  geom_vline(xintercept = c(365, 1460), linetype = "dashed")
+
+# Plot the mosquito population through time:
+simulation_output |>
+  mutate(pfpr210 = n_detect_lm_730_3650 / n_730_3650) |>
```

Comment:
This is a `dev` malariasimulation branch naming convention. I think we should stick with the main branch, which would make it `n_detect_730_3650` I think

Reply:
Changed!